test/e2e/_helpers.js

```diff
@@ -14,8 +15,30 @@ import personService from '../../common/services/person'
 import personEscortRecordService from '../../common/services/person-escort-record'
 import profileService from '../../common/services/profile'
 import referenceDataService from '../../common/services/reference-data'
+import { SENTRY } from '../../config'
 import { formatDate } from '../../config/nunjucks/filters'
 
+if (SENTRY.DSN) {
+  Sentry.init({
+    dsn: SENTRY.DSN,
+    environment: SENTRY.ENVIRONMENT,
+  })
+}
+
+function errorHandler(body) {
+  return err => {
+    Sentry.withScope(scope => {
+      err.errors.forEach((apiErr, idx) => {
+        scope.setContext(`error_${idx}`, apiErr)
+      })
+      scope.setContext('body', body)
+      Sentry.captureException(err)
```

Comment:
I'm not sure what level we want to capture these as: `error`, `warning`, `info`. The default is `error`.

They will be filtered out already by the `environment` value and won't create noise in the main alerts channel so maybe `error` is okay.

Reply:
Sounds reasonable

Reply:
Yeah, let's see how noisy `error` is, and work from there.